File: c-sharp/ws-client/model/events/OrderBookRawEvent.cs
namespace ws_client.model.events
{
    class OrderBookRawEvent
    {
        public string ChannelId { get; set; }
        public long Id { get; set; }
        public long Changed { get; set; }
        public decimal? Price { get; set; }
        public decimal Quantity { get; set; }

        public override string ToString() => base.ToString() +
            ": {" +
                "channelId='" + ChannelId + '\'' +
                ", id=" + Id +
                ", changed=" + Changed +
                ", price=" + Price +
                ", quantity=" + Quantity +
                '}';
    }
}

File: java/src/main/java/model/events/OrderBookRawEvent.java
package model.events;

import java.math.BigDecimal;

public class OrderBookRawEvent {
    private String channelId;
    private Long id;
    private Long changed;
    private BigDecimal price;
    private BigDecimal quantity;

    public String getChannelId() {
        return channelId;
    }

    public void setChannelId(String channelId) {
        this.channelId = channelId;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getChanged() {
        return changed;
    }

    public void setChanged(Long changed) {
        this.changed = changed;
    }

    public BigDecimal getPrice() {
        return price;
    }

    public void setPrice(BigDecimal price) {
        this.price = price;
    }

    public BigDecimal getQuantity() {
        return quantity;
    }

    public void setQuantity(BigDecimal quantity) {
        this.quantity = quantity;
    }

    @Override
    public String toString() {
        return "OrderBookRaw{" +
                "channelId='" + channelId + '\'' +
                ", id=" + id +
                ", changed=" + changed +
                ", price=" + price +
                ", quantity=" + quantity +
                '}';
    }
}
